### ISSUE TITLE
[Feature Request] Add auto-play trailer for homescreen widgets

### ISSUE THREAD

**Issue by rijsab** [SerpentDrago/skin.auramod#187]
Hi,

I cannot find an option for automatic trailer on home screen? Is this correct? Trailers are only playing in info dialogue but not on home screen for Netflix experience. 

Thanks.

**Comment by inb4after** [comment]
That is correct. The only auto-trailer options are for spotlights on homescreen and info screen as you mentioned.

**Comment by rijsab** [comment]
Thanks - is it possible to have this feature for home screen as well for all widgets?

**Comment by inb4after** [comment]
It is possible, but I'm not sure if it's recommended

**Comment by rijsab** [comment]
Cheers, I truly believe this will make the Netflix view more consistent.

**Comment by SerpentDrago** [comment]
This would slow the skin to a fucking crawl and prob go through youtube api request limit in short time . if you want a trailer just click the damn widget to pull up the netflix info page and click trailer .

I mean yeh it could be done .  but not advised .

**Comment by rijsab** [comment]
Relax.. not using the skin any more. Take some rest pal. Cheers

**Comment by forsajd1** [comment]
Hello: Could I add the option to deactivate clearlogo so that it is only in the text of the movie? Skin arctic zephyr reoladet so it has
Well thank you
